app/models/comment.rb

```diff
@@ -64,6 +75,12 @@ def initialize(person, target, text)
       super(person, target)
     end
 
+    def build(*args)
+      super.tap do |comment|
+        comment.filter_mentions(comment.people_allowed_to_be_mentioned)
```

Comment:
We only filter this on send? and not on receive? Or do I miss something? So what happens, if we receive a comment with a mention (maybe from another network that doesn't have this outgoing filter), and the mentioned user can not see the post? does he receive a notification?